MantidQt/CustomInterfaces/src/Tomography/TomoToolConfigDialogBase.cpp

```diff
@@ -0,0 +1,38 @@
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogBase.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogTomoPy.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogAstra.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogSavu.h"
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogCustom.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+TomoToolConfigDialogBase *
+TomoToolConfigDialogBase::getCorrectDialogForToolFromString(
```

Comment:
Could be better to return a `unique_ptr<TomoToolConfigDialogBase>` and remove the `new` call, and call `Kernel::make_unique` instead